lib/shared/permissions.js

```diff
@@ -73,6 +72,32 @@ exports.isElevated = () => {
 };
 
 /**
+ * @summary Quote and escape a string
+ * @function
+ * @private
+ *
+ * @param {String} string - input string
+ * @returns {String} quoted and escaped string
+ *
+ * @example
+ * const result = quoteAndEscapeString('foo');
+ * console.log(result);
+ *
+ * // Windows
+ * "foo"
+ *
+ * // Unix
+ * 'foo'
+ */
+const quoteAndEscapeString = (string) => {
+  if (os.platform() === 'win32') {
+    return `"${string}"`;
+  }
+
+  return `'${_.replace(_.toString(string), /'/g, '\\\'')}'`;
```

Comment:
Hmmm, perhaps it'd be better to have `string = _.toString(string);` as the first line of this function?